gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -169,9 +170,14 @@ def _bigquery_to_vcf_shards(
     variants = (p
                 | 'ReadFromBigQuery ' >> beam.io.Read(bq_source)
                 | bigquery_to_variant.BigQueryToVariant(annotation_names))
-    call_names = (variants
-                  | 'CombineCallNames' >>
-                  combine_call_names.CallNamesCombiner())
+    if known_args.sample_names:
```

Comment:
Please add a TODO to modify the SQL query to only return the specified call names to reduce the amount of data being processed from the beginning (I think the query would be more complex, so it's fine to leave as is for now).

Reply:
That's a good point! Add a TODO for now.